src/js/components/award/financialAssistance/CFDAOverview.jsx

```diff
@@ -5,28 +5,50 @@ import { TooltipWrapper } from 'data-transparency-ui';
 import { CFDAOverviewInfo } from '../shared/InfoTooltipContent';
 
 const propTypes = {
-    cfdaPropgram: PropTypes.string
+    cfdaPropgram: PropTypes.string,
+    cfdaCount: PropTypes.number,
+    jumpToSection: PropTypes.func
 };
 
-const CFDAOverview = ({
-    cfdaPropgram
-}) => (
-    <div className="award-overview__right-section__cfda award-overview-column first award-overview-column__spacing">
-        <h6 className="award-overview-title">
-            CFDA Program / Assistance Listing
-            <TooltipWrapper
-                className="award-section-tt"
-                icon="info"
-                left
-                tooltipComponent={CFDAOverviewInfo} />
-        </h6>
-        <div className="award-overview__body award-overview__cfda">
-            <span>
-                {cfdaPropgram}
-            </span>
-        </div>
-    </div>
-);
+export default class CFDAOverview extends React.Component {
```

Comment:
We can keep this a functional component buy moving the `jumpToCFDASection` to a variable. The code is below.

```import React from 'react';
import PropTypes from 'prop-types';
import { TooltipWrapper } from 'data-transparency-ui';

import { CFDAOverviewInfo } from '../shared/InfoTooltipContent';

const propTypes = {
    cfdaPropgram: PropTypes.string,
    cfdaCount: PropTypes.number,
    jumpToSection: PropTypes.func
};

const CFDAOverview = ({
    cfdaPropgram,
    cfdaCount,
    jumpToSection
}) => {
    const jumpToCFDASection = () => {
        jumpToSection('cfda');
    };
    return (
        <div className="award-overview__right-section__cfda award-overview-column first award-overview-column__spacing">
            <h6 className="award-overview-title">
                {cfdaCount === 1 ?
                    "CFDA Program / Assistance Listing" : "Primary CFDA / Assistance Listing"}
                <TooltipWrapper
                    className="award-section-tt"
                    icon="info"
                    left
                    tooltipComponent={CFDAOverviewInfo} />
            </h6>
            <div className="award-overview__body award-overview__cfda">
                <span>
                    {cfdaPropgram}
                </span>
                <div>
                    <button
                        key="cfda"
                        className="award-viz__button"
                        onClick={jumpToCFDASection}>
                        <div className="award-viz__link-text">
                            {cfdaCount === 1 ? "View more info on this program" : `View all ${cfdaCount} CFDA Programs`}
                        </div>
                    </button>
                </div>
            </div>
        </div>
    );
}

CFDAOverview.propTypes = propTypes;
export default CFDAOverview;```

Reply:
Running with that code gives a `Uncaught ReferenceError: React is not defined` error and the page fails to load.

Reply:
Interesting, it is working on my comp